recipes/r-pcse/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set version = '1.9.1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pcse
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pcse_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pcse/pcse_{{ version }}.tar.gz
+  sha256: c8a19f898d0395a92e9215c270b1b9f338a7ebf4d98d5dd836c34db90d0969e5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('pcse')"           # [not win]
+    - "\"%R%\" -e \"library('pcse')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=pcse
+  license: GPL-3
```

DESCRIPTION shows `License: GPL (>= 3)`, so

```suggestion
  license: GPL-3.0-or-later
```